### PR TITLE
[US12] Add connection to the H2 db and create user table.

### PR DESCRIPTION
**Pull request by AndriiHorbenko**
Created User Entity, added connection to H2db with the application.yaml

### CODE REVIEW

File: social_network_api/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,25 +1,25 @@
 spring.main.banner-mode=off
```
Comment (miegami) [inline social_network_api/src/main/resources/application.properties:1]: This file can be removed.

File: social_network_api/src/main/java/app/model/User.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package app.model;
```
Comment (miegami) [inline social_network_api/src/main/java/app/model/User.java:1]: Please, create package com.danit.socialnetwork

**Comment by miegami** [comment]
Please. resolve merge conflicts by merging main into your branch.

File: social_network_api/src/main/java/app/model/User.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+import lombok.Data;
+
+import javax.persistence.*;
```
Comment (miegami) [inline social_network_api/src/main/java/app/model/User.java:5]: Don't use wildcard in the imports.

File: social_network_api/src/main/java/app/model/User.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import java.time.LocalDate;
+
+@Entity (name = "users")
+@Data
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "user_id")
+    Integer user_id;
```
Comment (miegami) [inline social_network_api/src/main/java/app/model/User.java:13]: Please don't use _ in the namings. Java uses camelCase

File: social_network_api/src/main/resources/application.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    console:
+      enabled: true
+      path: /h2
+  main:
+    banner-mode: off
+
+  logging:
+    level:
+      root=ERROR:
+      org=info:
```
Comment (miegami) [inline social_network_api/src/main/resources/application.yaml:22]: ¯\_(ツ)_/¯

Comment (AndriiHorbenko) [reply social_network_api/src/main/resources/application.yaml:22]: I don`t understand for what it is, we did this to configure logging. It doesn`t bother me and I left not to forget the syntax

Comment (miegami) [reply social_network_api/src/main/resources/application.yaml:22]: I mean that yaml files don't understand '=' sign. Use ':' instead of it

**Comment by miegami** [comment]
And still red pipeline. Read the message in slack and check pipeline logs

**Comment by AndriiHorbenko** [comment]
> And still red pipeline. Read the message in slack and check pipeline logs

I need more clarification! I read in slack and did changes in IDE, but I am not familiar with pipeline log. Where can I see red pipeline?

**Review by miegami — APPROVED** [review]

**Comment by AndriiHorbenko** [comment]
Thanks! Got it! It was new for me!

**Review by AlexKhmarenko — APPROVED** [review]

**Pull request merged**